XSLT/utkMODStoMODS.xsl

```diff
@@ -1,282 +1,163 @@
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300015387">diagrams</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300033973">drawings (visual works)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300028881">ephemera (general)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300028048">filmstrips</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300015578">illustrations</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300028569">manuscripts (document
+            genre)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300028094">maps (documents)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300027440">minutes (administrative
+            records)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300037316">money</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300263857">moving images</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300127173">negatives (photographic)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300311889">objects</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300202595">oral histories (document
+            genres)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300033618">paintings (visual works)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300220572">pamphlets</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300026657">periodicals</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300046300">photographs</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300026816">postcards</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300027221">posters</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300264821">printed ephemera</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300041273">prints (visual works)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300247936">record covers</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300026685">records (documents)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300026427">scores</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300123013">signs (declatory or advertising
+            artifacts)</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300028633">sound recordings</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300127478">trasnparencies</f>
+        <f authority="aat" uri="http://vocab.getty.edu/aat/300028682">video recordings</f>
+    </xsl:param>
+
+    <!-- Drop Unknown Names -->
+    <xsl:template match="name[namePart[matches(text(), lower-case('unknown'))]]"/>
+
+    <!-- Apply Creators and Contributors -->
+    <xsl:template match="name[namePart[not(matches(text(), lower-case('unknown')))]]">
```

Comment:
Almost the same here `name[namePart[not(matches(lower-case(text()), 'unknown')))]].

Reply:
@CanOfBees, sorry about that.  Not sure what I was thinking.  Let me know if the latest push looks good.

Reply:
@markpbaggett [line 110](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/pull/59/files#diff-069218ac714e500f6905c223d868b556R110) looks fine, but [line 113](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/pull/59/files#diff-069218ac714e500f6905c223d868b556R113) has the same problem. Sorry - don't think I made that clear in the comments.